### ISSUE TITLE
[Ashwini Support] Update vaccination checklist

### ISSUE THREAD

**Issue by nupoorkhandelwal** [OpenCHS/ashwini#32]
org : Ashwini2
Support ticket - 
https://avni.freshdesk.com/a/tickets/2216

Vaccination schedule
https://docs.google.com/spreadsheets/d/1NDSWkPWZt22j-4m9kgfEqLaPx_lL1G8W76-vmabytp8/edit#gid=0

https://docs.google.com/spreadsheets/d/10AjhR7NUlOKlQogJk5x0sEG9KRmwmDCMbulbBdaK_QE/edit#gid=0

- [x] Update the checklist with the above mentioned schedule on UAT

**Comment by nupoorkhandelwal** [comment]
There are too many scenarios, did not completely test for each and every vaccine status, will suggest to ask Kannan to test it thoroughly once our internal testing is done.

**Comment by Dinesh2019** [comment]
![image.png](https://images.zenhubusercontent.com/640ef9e3ad2fcf337b550a3f/1c1b77d3-5b8f-4320-ac05-695603c9ec68)

**Comment by Dinesh2019** [comment]
Please refer the doc for inputs
https://docs.google.com/spreadsheets/d/1v8Jhacug51FEF93qipgM5ROqKbByBb8ES_UNpSG-RKA/edit#gid=0

**Comment by nupoorkhandelwal** [comment]
@Dinesh2019 please test the vaccines added with new individuals. not sure why some vaccines are not coming for existing individuals.

**Comment by Dinesh2019** [comment]
OPV 1 -- OPV1 instead of overdue it is calcualted as expired 
OPV 2 -- OPV2 instead of overdue it is calcualted as expired 
OPV 3 -- OPV3 instead of overdue it is calcualted as expired 
IPV 1 -- IPV1 instead of overdue it is calcualted as expired 
IPV 2 -- IPV2 instead of overdue it is calcualted as expired 
Measles 1/MR 1 -- MR 1 instead of overdue it is calcualted as expired 
Vit. A -- Vit.A is missing in the checklist
OPV Booster -- opv Booster  instead of overdue it is calcualted as expired 
Vitamin A  --  Insteaded of due it is calculated as overdue

**Comment by Dinesh2019** [comment]
- [ ] for Vitamin A we calculated as 442 days as due which falls as sep 19 2024  it is calculated as over due

**Comment by nupoorkhandelwal** [comment]
Moving this card to On Hold as we need exact duration for Expired. Right now it got missed in requirements gathering.

**Comment by Dinesh2019** [comment]
vaccine checklist is not updated as per the excepted result
1. Scenario: birth(0 days)
     BGC, Hep B, OPV 0 due birth, only polio 0 is displayed BGC and HepB is not calculated 
    Individual data : Test dinaug0801
Screenshot:
![polio.png](https://images.zenhubusercontent.com/640ef9e3ad2fcf337b550a3f/7d3569f3-fd05-47fc-923d-32d8e472757a)

2.Scenario : Rotavirus vaccine 1
due : 42 days crtical : 49 days 0verdue : 365 expired : 366
individual : test 42days dinaug0801
![42days.png](https://images.zenhubusercontent.com/640ef9e3ad2fcf337b550a3f/3b7dfe38-eef3-49cc-8fa3-2315b11a2466)

**Comment by nupoorkhandelwal** [comment]
@Dinesh2019 Please check

1. Check child - "0 days" - It has all the three  vaccines
2. Check child - "42 dayss" - It has Rota1

**Comment by Dinesh2019** [comment]
Test scenario:
1. created the individual and used the boundary value analysis method to test this eg: for the scenario, Rota 2 due date is 70 days then we can give the date as 69, 70,71
2. Edit the individual birth date to ensure the vacancies are updated as per the dates.
